### PR TITLE
Ping/bug fix

### PR DESCRIPTION
**Pull request by Awesomeman007**
- set jwt expiration date to 30 days (which I did earlier but not saved for some reason)
- specifically specifying 200 for login ok response
- throwing error instead of response in fail otp verification

### CODE REVIEW

File: src/otp/otp.service.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { HttpStatus, HttpException, Injectable } from '@nestjs/common'
+import { HttpStatus, HttpException, Injectable } from '@nestjs/common';
+import { UnauthorizedError } from 'type-graphql';
```
Comment (sad7898) [inline src/otp/otp.service.ts]: What's this import?

File: src/otp/otp.service.ts [hunk 2]
```diff
@@ -54,14 +55,10 @@ export class OtpService {
...
-    } else {
-      return {
-        status: "fail",
-        message: "Code is incorrect."
-      }
+        status: 'success',
+        message: 'Code is correct.',
+      };
     }
+    throw new HttpException('Failed Verifying OTP', HttpStatus.BAD_REQUEST)
```
Comment (patarapornkan) [inline src/otp/otp.service.ts:61]: change @ApiUnauthorizedResponse({ description: 'Failed Verifying OTP' }) in otp.controller to @ApiBadRequestResponse

File: src/otp/otp.service.ts (hunk 1 above)
Comment (Awesomeman007) [reply src/otp/otp.service.ts]: its auto import.

File: src/otp/otp.controller.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export class OtpController {
 
   @ApiBody({ type: OtpDTO })
   @ApiOkResponse({ description: 'Verified Successfully' })
-  @ApiUnauthorizedResponse({ description: 'Failed Verifying OTP' })
+  @ApiOkResponse({ description: 'Failed Verifying OTP' })
```
Comment (patarapornkan) [inline src/otp/otp.controller.ts:19]: change ApiOkResponse to ApiBadRequestResponse

Comment (patarapornkan) [inline src/otp/otp.controller.ts]: change ApiOkResponse to ApiBadRequestResponse

**Review by patarapornkan — APPROVED** [review]

**Pull request merged**